### PR TITLE
Add support for custom migration tasks

### PR DESCRIPTION
**Pull request by fmnoise**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -158,6 +158,8 @@ Rails.configure do |config|
 
   config.rails_db_views.functions_path += %w( /some/function/path )
   config.rails_db_views.functions_extension = "*.sql"
+
+  config.rails_db_views.migration_tasks += %w( db:migrate:with_data )
```
Comment (mathieujobin) [inline README.md:162]: why is this called `with_data` ?  i also don't see the code for this added task? 
everything below seems to be just an unrelated refactor.

Comment (fmnoise) [reply README.md:162]: this an is example in readme file - additional rake task which will be handled as db migration task. if you use `data-migrate` gem then you'll need to add this to your configuration

Comment (fmnoise) [reply README.md:162]: also commit message - "Add support for custom migration tasks" is pretty self-describing
